src/services/datastore.js

```diff
@@ -29,7 +29,7 @@ export async function queryDatastoreConfig() {
 export async function queryMonthIndices(params) {
   const { datastoreConfig } = params;
 
-  const endpoint = `${datastoreConfig.elasticsearch}/_cat/indices?format=json&pretty=true`;
+  const endpoint = `${datastoreConfig.elasticsearch}/_aliases`;
```

Comment:
Does `_aliases` give you JSON output?

Reply:
Yes, it returns JSON with pretty formatting by default.

Reply:
But our production V1 Elasticsearch instance returns an empty list because we don't use aliases.

Reply:
Okay, I can see how this might be more efficient, as it does return JSON and is a much smaller payload.